lib/gui/modules/analytics.js

```diff
@@ -39,6 +41,42 @@ const analytics = angular.module(MODULE_NAME, [
   require('../models/settings')
 ]);
 
+/**
+ * @summary Create an object copy with all absolute paths replaced with the path basename
+ * @function
+ * @private
+ *
+ * @param {Object} object - original object
+ * @returns {Object} transformed object
+ *
+ * @example
+ * const anonymized = hidePaths({
+ *   path1: '/home/john/rpi.img',
+ *   simpleProperty: null,
+ *   nested: {
+ *     path2: '/home/john/another-image.img',
+ *     path3: 'yet-another-image.img',
+ *     otherProperty: false
+ *   }
+ * });
+ *
+ * console.log(anonymized);
+ * > {
+ * >   path1: 'rpi.img',
+ * >   simpleProperty: null,
+ * >   nested: {
+ * >     path2: 'another-image.img',
+ * >     path3: 'yet-another-image.img',
+ * >     otherProperty: false
+ * >   }
+ * > }
+ */
+const hidePaths = (object) => {
+  return _.deepMapValues(object, (value) => {
```

Comment:
Not sure if its worth or not, but you could implement this recursively to avoid the extra dependency. For example: if the value of a property is an object, you run `hidePath` on it again.